### PR TITLE
296 test coverage on all things record hooks

### PR DESCRIPTION
**Pull request by madaley1**
Added some extrapolated tests, but to be honest I'm a bit at a loss on how else to test it. I'll pick it up in the morning and see if I can figure out some other tests to write other than testing valid and invalid values as those are testing the record methods more than they are the record hook.

### CODE REVIEW

File: plugins/record-hook/src/tests/bulkRecordHook.messages.e2e.spec.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    )
+    sheetId = workbook.sheets[0].id
+  })
+
+  afterAll(async () => {
+    await deleteSpace(spaceId)
+  })
+
+  describe('Assigns metadata without assigning a new value', () => {
+    it('correctly assigns metadata', async () => {
```
Comment (carlbrugger) [inline plugins/record-hook/src/tests/bulkRecordHook.messages.e2e.spec.ts:40]:
```suggestion
  describe('Assigns messages without assigning a new value', () => {
    it('correctly assigns messages', async () => {
```

File: plugins/record-hook/src/tests/recordHook.messages.e2e.spec.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    )
+    sheetId = workbook.sheets[0].id
+  })
+
+  afterAll(async () => {
+    await deleteSpace(spaceId)
+  })
+
+  describe('Assigns metadata without assigning a new value', () => {
+    it('correctly assigns metadata', async () => {
```
Comment (carlbrugger) [inline plugins/record-hook/src/tests/recordHook.messages.e2e.spec.ts:40]:
```suggestion
  describe('Assigns messages without assigning a new value', () => {
    it('correctly assigns messages', async () => {
```

File: plugins/record-hook/src/tests/recordHook.messages.e2e.spec.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      listener.use(
+        recordHook('test', (record) => {
+          record.addInfo('name', messageValue)
+        })
+      )
+      await createRecords(sheetId, defaultSimpleValueData)
+
+      await listener.waitFor('commit:created')
+      const records = await getRecords(sheetId)
+      console.dir(records[records.length - 2].values['name'].messages)
```
Comment (carlbrugger) [inline plugins/record-hook/src/tests/recordHook.messages.e2e.spec.ts:50]: debug code

File: plugins/record-hook/src/tests/bulkRecordHook.messages.e2e.spec.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          records.map((record) => {
+            record.addInfo('name', messageValue)
+          })
+        )
+      )
+      await createRecords(sheetId, defaultSimpleValueData)
+
+      await listener.waitFor('commit:created')
+      const records = await getRecords(sheetId)
+      console.dir(records[records.length - 2].values['name'].messages)
```
Comment (carlbrugger) [inline plugins/record-hook/src/tests/bulkRecordHook.messages.e2e.spec.ts:52]: debug code

**Review by carlbrugger — CHANGES_REQUESTED** [review]

**Comment by ashleygmulligan2** [comment]
@madaley1 - can you keep chugging along on this over the week? Not urgent necessarily, but don't want the work to get lost.

**Comment by carlbrugger** [comment]
@madaley1 I think this is ready to go once the conflict is resolved

**Comment by madaley1** [comment]
Hey @carlbrugger @ashleygmulligan2, this should be ready to ship whenever you get a moment to review!

**Pull request merged**